stretch-oracle-java8/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM debian:stretch
+MAINTAINER "Patrick Plocke" <patrick.plocke@flaconi.de>
+
+
+###
+### Build arguments
+###
+ARG JAVA_VERSION=8
+ARG ORACLE_REP="deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main"
+ARG ORACLE_KEY="c2518248eea14886"
```

Comment:
I'm strongly concerned about the legal implications of this. AFAIK, this is against Oracle's terms. Besides, why not simply go with OpenJDK? Are any of the apps that you use this image for not supporting it?

Reply:
There will be oracle as well as openjdk inside (openjdk coming afterwards). However for the sos scheduler I require the oracle version.

Reply:
I also added a link to the JDK's binary code license